### ISSUE TITLE
install.py create_virtualenv should use venv if it exists

### ISSUE THREAD

**Issue by kurtgraustein** [oracle/oci-cli#165]
If an approved Python3 installation already exists, install.py does not need to download and install virtualenv.  It can just use venv, instead.  The following are commands I used to edit v2.5.15 of install.py to replace/modify the contents of the create_virtualenv subroutine to do this:

$ sed -i "/^    download_location = os.path.join(tmp_dir, VIRTUALENV_ARCHIVE)$/,/^            shutil.copy(dll, dest_dir)$/d" install.py
$ sed -i "s/^    cmd = \[sys.executable, 'virtualenv.py', '--python', sys.executable, install_dir\]/    cmd = [sys.executable, '-m', 'venv', install_dir]/" install.py
$ sed -i "s/^    exec_command(cmd, cwd=working_dir)/    exec_command(cmd)/" install.py

**Comment by paul-hummel-oracle** [comment]
@kurtgraustein, thank you for trying this out and making the suggestion.
I found this thread pretty informative too:
https://stackoverflow.com/questions/41573587/what-is-the-difference-between-venv-pyvenv-pyenv-virtualenv-virtualenvwrappe

I think this would be beneficial, as sometimes folks have trouble downloading virtualenv.
We still support python 2 through the end of the year and would need virtualenv in that case.

Anyway, I will create an internal work item for this and update this thread as progress is made.

**Comment by kurtgraustein** [comment]
Thanks for the article link.  The following could be the new install logic:
```
if venv exists
  create virtual environment using venv
else
  if virtualenv does not exist
    download and install virtualenv
  end if
  create virtual environment using virtualenv
end if
```

**Comment by vish1** [comment]
@kurtgraustein This issue has been fixed as of release version 2.9.2.
